lib/streamlit/elements/lib/built_in_chart_utils.py

```diff
@@ -646,6 +649,10 @@ def _get_x_encoding(
         else:
             x_title = x_column
 
+    # User specified x-axis label takes precedence
+    if x_axis_label:
```

Comment:
nit: is there any valid use case for this to be an empty string lable? If so, it might be better to do `x_axis_label is None` so that its actually used if `""` ...  but not sure if someone wants to configure empty labels.